effects/effect.py

```diff
@@ -1,5 +1,5 @@
 # import logging
-from abc import ABC
+from abc import ABC, abstractmethod
```

Comment:
Code style:
Don't perform several imports in one line.
Separate them, please.
from abc import ABC
from abc import abstractmethod

https://www.python.org/dev/peps/pep-0008/#imports

Reply:
Automatic code formatter tells it is ok